_includes/sections/osoc.html

```diff
@@ -31,7 +31,7 @@ <h5> <span>01.</span> Configurator for individual route planners</h5>
             created for anyone who is interested in configuring their very own route planner to answer your personal
             mobility needs. We will demonstrate our configurator using the velomobile as a test case .
           </p>
-          <a href="">More informations</a>
+          <a href="">Details after the begining of Osoc, see you on 04/07</a>
```

Comment:
Switch to `OSoC`
Let's remove `href` attribute so it's not clickable :)
(same feedback goes for the following copies of this)

Reply:
Nevermind, I've seen #19 :)